charts/cp-kafka-rest/templates/deployment.yaml

```diff
@@ -20,6 +20,11 @@ spec:
   template:
     metadata:
       labels:
+      {{- if or .Values.podLabels .Values.prometheus.jmx.enabled }}
```

Comment:
Why does this depend on Prometheus being enabled?

Reply:
Good point, that happened to be in our specific use-case. 
I've removed the Prometheus enabled condition.